File: project_templates/server_openapi_generator/java/SecurityHandler.java
package {{#if project_info.package}}{{ project_info.package }}.{{/if}}securityHandlers;

import io.vertx.core.Handler;
import io.vertx.ext.web.RoutingContext;

public class {{ security_schema.class_name }} implements Handler<RoutingContext> {

    public {{ security_schema.class_name }}(){

    }

    @Override
    public void handle(RoutingContext routingContext) {
        // Handle {{ security_schema.schema_name }} security schema
        routingContext.next();
    }

}

File: project_templates/server_openapi_generator/kotlin/SecurityHandler.kt
package {{#if project_info.package}}{{ project_info.package }}.{{/if}}securityHandlers;

import io.vertx.core.Handler;
import io.vertx.ext.web.RoutingContext;

class {{ security_schema.class_name }}: Handler<RoutingContext> {

    override fun handle(routingContext: RoutingContext) {
        // Handle {{ security_schema.schema_name }} security schema
        routingContext.next()
    }

}